docs/public-networks/concepts/proof-of-stake.md

```diff
@@ -0,0 +1,49 @@
+---
+description: Ethereum proof of stake
+---
+
+# Proof of stake
+
+[The Merge](the-merge.md) transitions Ethereum Mainnet to
+[proof of stake (PoS)](https://ethereum.org/en/developers/docs/consensus-mechanisms/pos/) consensus.
+
+In Ethereum's PoS, you must run a [full node](the-merge.md#execution-and-consensus-clients) and
+[stake 32 ETH](https://ethereum.org/en/staking/) to become a validator.
+
```

Comment:
As it's the conceptual sections, I would "duplicate" some of the text in [the PoS explanation link](https://ethereum.org/en/developers/docs/consensus-mechanisms/pos/#what-is-pos) to explain the goal of PoS without having to leave the doc to read the info. We can probably write a shorter, more efficient version of it.